gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -354,6 +356,13 @@ message TripUpdate {
     // populated and will be ignored by consumers.
     // NOTE: This field is still experimental, and subject to change. It may be formally adopted in the future.
     optional string start_time = 3;
+    // Specifies the new shape of the vehicle travel path when the trip shape differs from the shape specified in
```

Comment:
The way it's worded, it seems to only apply to trips that have a linked/original shape_id in the CSV files.

That's ambiguous for trips with scheduleRelationship = ADDED because they have no linked shape_id in the CSV files.

What's the behaviour of this optional field when it's missing and there is no original CSV shape_id?
 
We can maybe clearly specify that the shape is unknown if no shape_id is provided for trips without linked/original shape_id? (rather than forcing the producer to always include the shape_id, that they might not be able to due to technical limitations)

Reply:
Great point! I've updated the language in-line to help reduce ambiguity there as well as explicitly specify the not provided scenario.